jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
ok, I am not sure that is right. The problem is the `Debug` build is quite different from `Release` builds. I know that in iOS, the release builds will not generate some extra files to debug purpose for example. I think the configuration should be specified in the `xcodeBuild` step. We have that info passed in from millicore, is there a reason we can't use it?